deploy/test/test_cases/TEST_ID_25_helm_default_retry_successful.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -euxo pipefail
+
+# This test verifies that default values for retry mechanism work
+setup_helm_environment
+
+pushd ../../
+  # Install HELM with a K8s Secret that does not exist
+  export LABELS="app: test-helm"
+  export DEBUG="true"
+  # A parameter that will force a failure
+  export CONJUR_APPLIANCE_URL="https://conjur-follower.${CONJUR_NAMESPACE_NAME}.svc.cluster.local/xyz"
+  fill_helm_chart_no_override_defaults
+  helm install -f "../helm/secrets-provider/ci/take-defaults-test-values.yaml" \
+    secrets-provider ../helm/secrets-provider \
+    --set-file environment.conjur.sslCertificate.value="test/test_cases/conjur.pem"
+popd
+
+pod_name=$($cli_with_timeout get pods --namespace=$APP_NAMESPACE_NAME --selector app=test-helm --no-headers | awk '{print $1}' )
+# Find initial authentication error that should trigger the retry
+$cli_with_timeout "logs $pod_name | grep 'CSPFK010E Failed to authenticate'"
+# Start the timer for retry interval
+start=$SECONDS
+
+echo "Expecting Secrets Provider retry configurations to take defaults RETRY_INTERVAL_SEC 30 and RETRY_COUNT_LIMIT 3"
+$cli_with_timeout "logs $pod_name | grep 'CSPFK010I Updating Kubernetes Secrets: 1 retries out of 3'"
+
+duration=$(( SECONDS - start ))
+echo $duration
+if (( 25 <= $duration <= 35 )); then
```

Comment:
I think it will be better to work with variable for RETRY_INTERVAL_SEC and RETRY_COUNT_LIMIT instead of (30,3)
If we will do some changes in the default values i dont want to remember go to tests as well. It suppose to be handle automatic.
Instead of 25 and 35 i would say for example RETRY_COUNT_LIMIT / 100 *80 (80%) , RETRY_COUNT_LIMIT / 100 *120 (120%) , by that this test will be more stable.